.github/workflows/coverage.yml

```diff
@@ -12,23 +12,28 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: true
-
-      - uses: dtolnay/rust-toolchain@master
+      - uses: dtolnay/rust-toolchain@nightly
+        with:
+          components: rustc, rust-std, cargo, llvm-tools, llvm-tools-preview
+      - uses: baptiste0928/cargo-install@v3
         with:
-          toolchain: nightly
+          crate: grcov
```

Comment:
👍🏻 makes perfect sense switching this to `nightly` instead of `master`. But then why bring in a separate action for the installer?

```yaml
       - run: cargo install grcov
```

should do the same thing, more simply.